src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -24,8 +24,8 @@ struct ReduceAxesParam : public dmlc::Parameter<ReduceAxesParam> {
     DMLC_DECLARE_FIELD(axis).set_default(TShape())
       .describe("The axis or axes along which to perform the reduction. "
                 "The default, `axis=()`, will compute over all elements into a "
-                "scalar array with shape `(1,)`.\n\nIf axis is int, "
-                "a reduction is performed on a particular axis.\n\n"
+                "scalar array with shape `(1,)`. If axis is int, "
```

Comment:
In "if axis is an int", "axis" refers to a function parameter is this correct? If so, I think i should show up in single backticks.